vassal-app/src/main/java/VASSAL/build/module/Map.java

```diff
@@ -3079,11 +3128,13 @@ public HelpFile getHelpFile() {
       Resources.getString(Resources.TOOLTIP_TEXT),
       Resources.getString(Resources.BUTTON_ICON),
       Resources.getString(Resources.HOTKEY_LABEL),
+      Resources.getString("Editor.Map.show_key"),
+      Resources.getString("Editor.Map.hide_key"),
```

Comment:
A few interesting things, most which are not new, but might be nice to nail now:

1. For clarity, I think in this case it might be better to change the 'Hotkey' label to 'Hotkey to show/hide':
![image](https://user-images.githubusercontent.com/1622667/114697330-23074680-9d61-11eb-8e1f-97a841ed46f6.png)

2. I set up this window as above, then deselected the 'Include toolbar to show/hide?' option, saved and reloaded the module. No button and the F5/F6/F7 definitions are no longer visible in the editor, but the F5/F6/F7 to toggle/show/hide still work! [F5 doesn't work in the Editor, but it does in the Player]. I think it is actually better than them not working. I'm thinking that the 3 Hotkey fields should be always visible, since they are working anyway.

3. While you're in there, how about fixing that thing where changing the 'Include toolbar to show/hide?' button doesn't actually change the visibility until the next time you reload the module. That is just painful, let's just make it a standard visibility change.

Reply:
Okay - I think latest set of commits (aka "BREAK that app! BREAK that app!") should do it. There was a deep rabbit hole of special cases but I think I have it largely sorted (and the "docked" state of a "first map window" in that category of things will still not change until restart -- I have updated messaging appropriately). A good thing is I think in the process I managed to clean up a few minor "bad behaviors". See what you think.